lib/gui/components/update-notifier/styles/_update-notifier.scss

```diff
@@ -20,3 +20,7 @@
   margin-bottom: 0;
 }
 
+.modal-update-notifier .modal-body a {
```

Comment:
Lets move the anchor style to `modal/styles/_modal.scss` and we should be good to merge.